en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc

```diff
@@ -297,23 +297,93 @@ To enable email notifications:
 . Toggle the button to enable email notifications.
 . Click *Save*.
 
+[[connect-composer-to-anypoint-platform]]
 == Connect Composer to Anypoint Platform
 
 Organizations that use both MuleSoft Composer and Anypoint Platform can connect the two products.
 
 To link Composer to Anypoint Platform:
 
-* Log in to Anypoint Platform.
-* In the navigation bar or the main Anypoint Platform page, click *Access Management*.
-* In the xref:access-management::managing-your-account.adoc[Access Management] navigation menu, click *Composer Sync*.
-* In the *Composer Sync Orgs* page, click *Add Composer Orgs*.
-* In the *Add organization* window, in the *Organization ID* box, enter your Composer organization and then click *Add*.
-+
-[NOTE]
+. In MuleSoft Composer, in the left pane, click *Settings* > *Account*. +
```

Comment:
"left" is not an accessible term. Is there another way to say this?

Reply:
I will fix this!